assets/javascripts/overview.js

```diff
@@ -287,8 +287,34 @@ function showAddCommentsDialog() {
   modal.show();
 }
 
-function addComments(form) {
+function restartJobs(btn) {
+  const form = btn.closest('form');
   const text = form.elements.text.value;
+  const jobs = btn.dataset.jobs
+    .split(',')
+    .map(Number)
+    .filter(n => !isNaN(n));
+  const apiUrl = btn.dataset.url;
+  if (!text.length) {
+    return window.alert("The comment text mustn't be empty.");
+  }
+
+  const progressIndication = document.getElementById('add-comments-progress-indication');
+  const controls = document.getElementById('add-comments-controls');
+  progressIndication.style.display = 'flex';
+  controls.style.display = 'none';
+  restartJob(apiUrl, jobs, text);
+  setTimeout(() => {
+    progressIndication.style.display = 'none';
+    controls.style.display = 'inline';
+    window.addCommentsModal.hide();
+  }, 500);
```

Comment:
Why not do this immediately? It would actually be bad if the user would accidentally click on the controls twice so I suggest to avoid the `setTimeout(…)` and just change the page state immediately. But also read my other comment of keeping the model open for the sake of having a progress indication.

Reply:
timeout removed